.github/workflows/create-additional-action-tags.yml

```
@@ -16,7 +16,9 @@ jobs:
           TAG=${GITHUB_REF/refs\/tags\//}
           MAJOR="v$(echo ${TAG} | cut -d. -f1)"
           MINOR="${MAJOR}.$(echo ${GITHUB_REF} | cut -d. -f2)"
-          echo "tags=${MAJOR}%0A${MINOR}" >> $GITHUB_OUTPUT
+          echo "tags=${MAJOR}<<EOF" >> $GITHUB_OUTPUT
+          echo "${MINOR}" >> $GITHUB_OUTPUT
+          echo "EOF" >> $GITHUB_OUTPUT
```

I think this should look like this:
```suggestion
          echo "tags<<EOF" >> $GITHUB_OUTPUT
          echo "${MAJOR}" >> $GITHUB_OUTPUT
          echo "${MINOR}" >> $GITHUB_OUTPUT
          echo "EOF" >> $GITHUB_OUTPUT
```